chemprop/args.py

```diff
@@ -97,6 +97,19 @@ class CommonArgs(Tap):
     """
     atom_descriptors_path: str = None
     """Path to the extra atom descriptors."""
+    overwrite_default_atom_descriptors: bool = False
+    """
+    Overwrites the default atom descriptors with the new ones instead of concatenating them.
+    Can only be used if atom_descriptors are used as a feature.
+    """
+    no_atom_descriptor_scaling: bool = False
+    """Turn off atom feature scaling."""
+    bond_descriptors_path: str = None
+    """Path to the extra bond descriptors that will be used as bond features to featurize a given molecule."""
+    overwrite_default_bond_descriptors: bool = False
+    """Overwrites the default atom descriptors with the new ones instead of concatenating them"""
+    no_bond_descriptor_scaling: bool = False
+    """Turn off atom feature scaling."""
```

Comment:
Arguments in the CommonArgs are supplied newly for predictions, overwriting what was used in training. If you need for the same argument to be used in the prediction as was used in the original training, then that argument should be specified in TrainArgs. The argument for `bond_descriptors_path` should be in CommonArgs, but I think the rest of these are things that need to be held consistent. To that point, I think it likely that `number_of_molecules`, `no_features_scaling`, and _possibly_ `features_generator` should be moved to TrainArgs.

Reply:
Interesting, I never thought about this before. I moved my new arguments as appropriate. But did not touch the other ones yet not to break anything, but this could be done in another pull request!

Reply:
I think worth noting again that this would make predictions from old checkpoints no longer work. I think it's the right way to specify these, but it will take some extra code to keep the back compatibility. I can write it if you think it's important and want the input.

Reply:
Maybe in that case we should just pay attention it is done the right way for the new arguments...